railgun/cmd/rootcmd/rootcmd.go

```diff
@@ -0,0 +1,29 @@
+package rootcmd
+
+import (
+	"context"
+	"flag"
+
+	"github.com/kong/kubernetes-ingress-controller/railgun/manager"
+	"github.com/spf13/cobra"
```

Comment:
nitpick, but I feel we should stick to the `goimports -local` style imports:

```suggestion
	"github.com/spf13/cobra"

	"github.com/kong/kubernetes-ingress-controller/railgun/manager"
```

Reply:
Fine by me either way - with `-local` or without. Our codebase doesn't seem consistent about this today. Becoming opinionated here is a code style change that requires reformatting the entire codebase, and whichever style we choose to maintain, it will need to be enforced by CI, otherwise it will rot.

If you feel strongly about this proposal, I recommend that we tackle this outside of the context of this PR: record an issue, reformat the codebase and add a CI check. WDYT?

Reply:
Sounds good, we can just run `goimports` across all files :+1: